Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -322,7 +322,10 @@ extension AlertManager: AlertIssuer {
     }
 
     private func replayAlert(_ alert: Alert) {
-        handlers.forEach { $0.issueAlert(alert) }
+        // Only alerts with foreground content are replayed
+        if alert.foregroundContent != nil {
+            handlers.forEach { $0.issueAlert(alert) }
+        }
```

Comment:
hm... I'm not sure this is always true. Repeating alerts need to be replayed, right?

Reply:
Also, I think this is already be handled.

Reply:
This is more a state management thing. Remember we have two sides to "notifications"; the system side, and the in-app modal. When Loop is restarted, the presentation state for the in-app side is lost, and needs to be "replayed".  It's not a re-issuance of any sort.  The iOS side is still fine; it doesn't lose state across a Loop restart, and uses system managed persistence for phone restarts. Let me know if you still have questions about this.

Reply:
It might be more clear if we avoid the call to "issueAlert" and replace it with a fixed call to foreground presenter, to avoid the code reading like it's re-issuing the alert. It's really just recovering the presentation state.

Reply:
So, if there is `foregroundContent` the user notification should be issued again as well? Should it be that only if there is `foregroundContent` that only the in-app portion should be reissued?

Reply:
Yes, I believe so. The intent of "replaying" alerts is to ensure that the user acknowledges alerts that require acknowledgement. I think the current behavior of also re-issuing them as system notifications is not correct, as that's not necessary just to ensure acknowledgement. They were already issued, and will still exist as a system managed user notification, unless the user has dismissed them, in which case they have already seen them, and have been notified.  I will take a look at how to best update the existing replay mechanism to just do this.

Reply:
OK. Will that be in another ticket or with this code?

Reply:
Added to this ticket.